sklearn/feature_selection/univariate_selection.py

```diff
@@ -232,12 +232,12 @@ def f_regression(X, y, center=True):
     Quick linear model for testing the effect of a single regressor,
     sequentially for many regressors.
 
-    This is done in 3 steps:
+    This is done in 2 steps:
 
-    1. The regressor of interest and the data are orthogonalized
-       wrt constant regressors.
-    2. The cross correlation between data and regressors is computed.
-    3. It is converted to an F score then to a p-value.
+    1. The cross correlation between data and regressors is computed
```

Comment:
You can either change this to

1. The cross-correlation between each regressor and the target is computed that is `((X - mean_X) * (y - mean_y)) / (std(X) * std(y))` or
2. The cross-correlation between each regressor and the target is computed by orthogonalizing the target and regressor of interest with a constant regressor and then dividing by the standard deviation of the target and the regressor.

I would prefer the first, since the second is unnecessarily verbose